utils.py

```diff
@@ -97,3 +99,121 @@ def get_details_from_id(alert_id):
 
     return retVal
 
+def parse_details_to_file_bug(details, oldest_alert):
+    branch = details['branch']
+    test = details['test']
+    platform = details['platform']
+    percent = details['percent']
+    push_date = details['push_date']
+    summary = ''
+
+    #add regression values
+    min_percent = min(percent)
+    max_percent = max(percent)
+    min_percent = re.findall('(?P<number>\d+)', min_percent)[0]
+    max_percent = re.findall('(?P<number>\d+)', max_percent)[0]
+    if int(max_percent) - int(min_percent) <= 1:
+        summary = summary + '%s' %(max_percent) + '% '
+    else:
+        summary = summary + '%s-%s' %(min_percent, max_percent) + '% '
+
+    #add platform
+    flags = {
+        'linux64': 0,
+        'linux32': 0,
+        'Win8 x64': 0,
+        'WinXP x32': 0,
+        'Win7 x32': 0,
+        'Android': 0,
+        'MacOS': 0
+    }
+    for p in platform:
+        if 'Ubuntu HW 12.04 x64' in p:
+            flags['linux64'] = 1
+        elif 'Ubuntu HW 12.04' in p:
+            flags['linux32'] = 1
+        elif 'WINNT 6.2 x64' in p:
+            flags['Win8 x64'] = 1
+        elif 'WINNT 5.1 (ix)' in p:
+            flags['WinXP x32'] = 1
+        elif 'WINNT 6.1 (ix)' in p:
+            flags['Win7 x32'] = 1
+        elif 'Mac' in p:
+            flags['MacOS'] = 1
+        elif 'Android' in p:
+            flags['Android'] = 1
+    add = ''
+    try_platform = ''
+    if flags['linux64'] and flags['linux32']:
+        add = add + 'Linux*/'
+        try_platform = 'linux,'
+    elif flags['linux64']:
+        add = add + 'Linux 64/'
+        try_platform = try_platform + 'linux64,'
+    elif flags['linux32']:
+        try_platform = try_platform + 'linux,'
+        add = add + 'Linux 32/'
+    if  (flags['Win8 x64'] and flags['WinXP x32']) or (flags['Win8 x64'] and flags['Win7 x32']):
+        try_platform = try_platform + 'win64,win32,'
+        add = add + 'Win*/'
+    elif flags['Win8 x64']:
+        try_platform = try_platform + 'win64,'
+        add = add + 'Win8 x64/'
+    elif flags['WinXP x32']:
+        try_platform = try_platform + 'win32,'
+        add = add + 'WinXP x32/'
+    elif flag['Win7 x32']:
+        try_platform = try_platform + 'win32,'
+        add = add + 'Win7 x32/'
+    if flags['MacOS']:
+        try_platform = try_platform + 'macosx64,'
+        add = add + 'MacOS*/'
+    if flags['Android']:
+        try_platform = try_platform + 'android-api-11,'
+        add = add + 'Android/'
+
+    try_platform = try_platform[:-1]
```

Comment:
I would prefer .strip(','), and likewise for add, .strip('/'), the [:-1] is not good for future editing.